test/testAll.m

```diff
@@ -46,6 +46,15 @@
 profile on;
 
 if ~isempty(strfind(getenv('HOME'), 'jenkins'))
+
+    % ignore list of files
+    ignoreFiles = {'./src/fluxomics/c13solver/IsotopomerModel.txt',
```

Comment:
Could you read from the `.gitignore` file instead of creating this list of ignored files?